browser/brave_browser_main_parts_mac.mm

```diff
@@ -7,11 +7,21 @@
 
 #include "brave/browser/mac/keystone_glue.h"
 #include "brave/browser/sparkle_buildflags.h"
+#include "brave/browser/updater/buildflags.h"
 
 #if BUILDFLAG(ENABLE_SPARKLE)
 #import "brave/browser/mac/sparkle_glue.h"
 #endif
 
+#if BUILDFLAG(ENABLE_OMAHA4)
+#include "base/task/task_traits.h"
```

Comment:
none of these headers should be inside `ENABLE_OMAHA4` except for `brave/browser/updater/features.h`

Reply:
Fixed in be0c977143dbf3092ea4dc035585546a61ddfc90.